src/API/QueryFieldService.php

```diff
@@ -199,4 +186,17 @@ private function loadFieldDefinition(Content $content, string $fieldDefinitionId
 
         return $fieldDefinition;
     }
+
+    /**
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
+     */
+    private function executeQueryAndMapResult(Query $query): array
+    {
+        return array_map(
+            function (SearchHit $searchHit) {
```

Comment:
Can be static:
```suggestion
            static function (SearchHit $searchHit) {
```
and it's missing return type (which I think can be established here, it's not `mixed`?)

Reply:
Fixed.